src/wp-admin/includes/class-wp-posts-list-table.php

```diff
@@ -1984,22 +1984,47 @@ public function inline_edit() {
 					<div class="inline-edit-group wp-clearfix">
 
 						<label class="inline-edit-status alignleft">
-							<span class="title"><?php _e( 'Status' ); ?></span>
+							<span class="title"><?php __( 'Status' ); ?></span>
```

Comment:
It will not print the title.

```suggestion
							<span class="title"><?php _e( 'Status' ); ?></span>
```